spec/factories/quotations.rb

```diff
@@ -7,7 +7,7 @@
 
   trait :tag do
     after(:build) do |quotation|
-      quotation.tags << create(:tag, name: 'Roman')
+      quotation.tags << create(:tag, name: "Roman#{random_id_generator}")
```

Comment:
Why not make the `:name` attribute on the tag factory a sequence?

Reply:
Done.

Personally I must say that I find it a bit ugly/involved to combine the sequences with the traits:

```ruby
  sequence :roman_name do |n|
    "RomanName#{n}"
  end

  trait :tag do
    after(:build) do |quotation|
      quotation.tags << create(:tag, name: "#{generate :roman_name}")
    end
  end
```

But. Whatevs. It's a more proper way of doing it. I hear and agree with your general points you wrote elsewhere regarding the dangers of using my own `random_id_generator` as a custom sequencer.

Reply:
Now updated further per our conversation yesterday.